tests/test_diffraction_objects.py

```diff
@@ -99,16 +84,16 @@
     (  # Different scat_quantity
         {
             "scat_quantity": "x-ray",
-            "wavelength": None,
```

Comment:
since we are testing the user behavior, it is most likely that User wouldn't enter `none` right?